### DISCUSSION TITLE
PSRAM ID read error: 0xffffffff, Failed to init external RAM!

### DISCUSSION THREAD

**Discussion by Antonderlange** [jomjol/AI-on-the-edge-device#1595] [Q&A]
(893) psram: This chip is ESP32-D0WD
E (894) psram: PSRAM ID read error: 0xffffffff
E (894) cpu_start: Failed to init external RAM!

will it work ?

**Comment by jomjol** [comment]
I assume no.

**Reply by caco3** [reply]
Search in the discussions, for this error, many others have it as well. It means that your hardware is bad.

See also https://github.com/jomjol/AI-on-the-edge-device/wiki/Error-Codes#0x00000001-psram-bad

**Reply by atsju** [reply]
New link : https://jomjol.github.io/AI-on-the-edge-device-docs/Error-Codes/

**Comment by atsju** [comment]
I will not work, PSRAM is dead. buy a new board

@jomjol  I did a some tests and in my case, it seems my "dead" board are not dead.
I did cross tests and the result of test depends on motherboard. 
Motherboard with only one button makes the PSRAM fail : 
![image](https://user-images.githubusercontent.com/4628382/212975415-0b13dd95-d3bb-4e64-86dc-b5bc37e7c038.png)
With 2 buttons like it works
![image](https://user-images.githubusercontent.com/4628382/212975563-0d5cf8d2-1241-405a-aceb-4aeb0fd80e5b.png)

Maybe this is the issue for some users... 
maybe faulty PSRAM for other users : https://github.com/jomjol/AI-on-the-edge-device/issues/1295#issuecomment-1362784330
